src/Files.App/Views/Shells/BaseShellPage.cs

```diff
@@ -135,8 +135,10 @@ public bool IsCurrentInstance
 					_IsCurrentInstance = value;
 
 					if (!value && SlimContentPage is not ColumnViewBrowser)
+					{
 						ToolbarViewModel.IsEditModeEnabled = false;
-
+						ToolbarViewModel.IsCommandPaletteOpen = false;
```

Comment:
Whenever `IsEditModeEnabled` is false, `IsCommandPaletteOpen` should also be false. There are other places where `IsCommandPaletteOpen` should be set to false.
How about modifying `IsEditModeEnabled` property setter so that if `IsEditModeEnabled` is set to false, `IsCommandPaletteOpen` is also set to false?